invoices.go

```diff
@@ -0,0 +1,95 @@
+package scalingo
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"gopkg.in/errgo.v1"
+)
+
+type InvoicesService interface {
+	ListInvoices(context.Context, PaginationOpts) (Invoices, PaginationMeta, error)
```

Comment:
From what I remember this is how we name the list functions in go-scalingo.

nitpick:
```suggestion
	InvoicesList(context.Context, PaginationOpts) (Invoices, PaginationMeta, error)
```